src/pages/gateway/command-reference.md

```diff
@@ -7,10 +7,48 @@ description: A description of the CLI commands available for API Mesh for Adobe
 
 The API Mesh for Adobe Developer App Builder CLI allows you to manage and modify meshes. This page covers commands exclusive to the API Mesh. For authorization and other Adobe I/O Extensible CLI commands, refer to the [Adobe IO CLI command list]. For installation instructions, refer to [Getting Started].
 
+## aio api-mesh:init
+
+Creates a [local development environment](./developer-tools.md#initiate-a-local-environment). You only need to use this command if you want to run meshes in your local environment. This command only needs to be run once.
```

Comment:
Lets reword this because we do not have a way to run meshes locally right now. It is slated for phase 2. Right now the advantage people get with creating a local environment is mesh maintenance and future features.